### PR TITLE
Rename AltTree -> AltBlockTree

### PR DESCRIPTION
**Pull request by Warchant**

### CODE REVIEW

File: docs/pypopminer.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+$ cd /tmp/bitcoin_func_test_uqgiab8_
+/tmp/bitcoin_func_test_qbjor7qu » ls -la
+total 8
+drwx------  6 bogdan bogdan  140 лип 22 23:17 .
+drwxrwxrwt 24 root   root   1040 лип 22 23:17 ..
+drwxr-xr-x  5 bogdan bogdan  120 лип 22 23:17 node0
+drwxr-xr-x  5 bogdan bogdan  120 лип 22 23:17 node1
+drwxr-xr-x  5 bogdan bogdan  120 лип 22 23:17 node2
+drwxr-xr-x  5 bogdan bogdan  120 лип 22 23:17 node3
+-rw-r--r--  1 bogdan bogdan 4298 лип 22 23:18 test_framework.log
```
Comment (blackyblack) [inline docs/pypopminer.md:69]: Try to avoid personal information and unicode characters in the doc.

File: docs/pypopminer.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# pypopminer {#pypopminer}
+
+`pypopminer` is a Python3 package, generated by `boost-python`, which contains MockMiner - data structure, which can mock out VeriBlock + Bitcoin + APM (Altchain POP Miner) completely. 
+
+Use it to generate endorsements of VBK in BTC, and ALT in VBK; generate VBK/BTC blocks with or without endorsements.
+
+### 0. Install dependencies
+
+`pypopminer` depends on [`boost-python`](https://www.boost.org/doc/libs/1_74_0/libs/python/doc/html/building.html), `python3-dev`.
+
```
Comment (blackyblack) [inline docs/pypopminer.md:10]: Instructions for specific OS eg Win, Linux, would be very helpful. I'd like to install dependencies with a copy-paste from the doc, not guessing them.

File: README.md [hunk 3]
```diff
@@ -4,10 +4,6 @@
...
 
 POP overall consists of 2 protocols:
-- POP Fork Resolution
-- POP Rewards
-
-[TOC]
-
-## POP Fork Resolution
+- POP Fork Resolution - given set of POP publications sent to Altchain from 2 chains, select best chain among these two, based on timeliness of publications in VBK tree, number of publications and other factors.
+- POP Rewards - an economic incentive for POP miners to send endorsing VBK transactions in VBK blockchain, and sending back proofs to Altchain. POP Rewards are paid in Altchain coins.
```
Comment (blackyblack) [inline README.md:8]: Proofreading from a native speaker would improve this text for sure.

File: docs/altblocktree.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+BtcChainParamsMain btcp;
+// your implementation of PayloadsProvider
+PayloadsProviderImpl provider;
+AltBlockTree tree(altp, vbkp, btcp, provider);
+```
+
+After initialization AltBlockTree does not contain any blocks. 
+**Users MUST bootstrap AltBlockTree** - add initial/genesis/root block of Altchain.
+
+We will refer to this block as **bootstrap block**.
```
Comment (blackyblack) [inline docs/altblocktree.md:24]: I think we support multiple bootstrap blocks.

File: docs/altblocktree.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+Bootstrap block is a first block in Altchain that can be endorsed, but can not contain POP body (PopData).
+
+If `bootstrap()` fails, your block provided in struct AltChainParams::getBootstrapBlock().
```
Comment (blackyblack) [inline docs/altblocktree.md:27]: This sentence looks unfinished.

File: docs/altblocktree.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+        return false;
+    }
+    
+    // then, attach block body to block header
+    tree.acceptBlock(block.getHash(), block.getPopData());
+    
+    // ...
+``` 
+
+AltBlockTree::acceptBlock() does connects block immediately if all previous blocks are connected, or just adds block body to AltBlock when one of previous blocks is not connected.
```
Comment (blackyblack) [inline docs/altblocktree.md:58]: Remove "does"

File: docs/altblocktree.md (hunk 4 above)
Comment (Warchant) [reply docs/altblocktree.md:24]: In Alt tree - only one.

**Pull request merged**